components/presenters/Docs/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { Badge } from '.'
+
+const stories = storiesOf('Badge', module)
+
+stories.add('Default', () => <Badge>3.0.0</Badge>)
+
+stories.add('Dark', () => <Badge variant="dark">3.0.0</Badge>)
```

Comment:
Rather than passing these strings around it can be nice to have the `const` as we have in other places, e.g.

```
const BADGE_VARIANT = {
  DARK: 'dark',
  LIGHT: 'light',
} as const
```
```